ports/xlsxio/vcpkg.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "xlsxio",
+  "version-string": "2021-03-24+0100",
```

Consider using `version-date` instead:
```suggestion
  "version-date": "2021-03-24",
```

---

Is there a default time zone?
Is `version-date` something like a hint?

---

There is not a default time zone, but this is not intended to be that granular. It is really only intended for ordering and rough suggestion of when the port was last changed.